docs/storage/upload/upload_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"strings"
```

Comment:
Great plan, I actually had that in but figured no other tests were doing that so perhaps the purpose of the example was not to be verbose? I'll add that in :)